masonite/middleware/VerifyEmailMiddleware.py

```diff
@@ -0,0 +1,17 @@
+''' Verify Email Middleware '''
+from masonite.request import Request
+
+class VerifyEmailMiddleware:
+    ''' Verify Email Middleware '''
+
+    def __init__(self, request: Request):
+        self.request = request
+
+    def before(self):
+        user = self.request.user()
+        if(user.verified_at is None){
+            self.request.redirect('/email/verify')
+        }
+
+    def after(self):
+        pass
```

Comment:
Just need to add the better docstrings to this too

Reply:
I actually just need to remove this file. It should just be in the Masonite repo right.